app/views/decidim/initiatives/initiative_votes/update_buttons_and_counters.js.erb

```diff
@@ -5,7 +5,7 @@
       userIdentitiesDialog = $('#user-identities'),
       refreshUrl;
 
-  morphdom($initiativeVotesCount[0], '<%= j(render partial: 'decidim/initiatives/initiatives/votes_count', locals: { initiative: current_initiative }) %>');
+  $initiativeVotesCount.html('<%= j(render partial: 'decidim/initiatives/initiatives/votes_count', locals: { initiative: current_initiative }) %>');
```

Comment:
Just out of curiosity: Is `morphdom` causing issues here? It's generally useful to mutate state with it, as it won't replace the whole dom (thus being more performant and also preventing breaking micro-states like `hover`, etc).

Reply:
As it was implemented morphdom was causing the issue. 

Anyway this patch must be interpreted in the current project context. It's a short week and we want to deliver an stable version ASAP. 

I agree with you that morphdom has many advantages over changing the entire dom. Maybe for an upcoming version we could invest more time and try to make it work using morphdom.

Reply:
No worries, just wanted to know if you figured out the exact issue or resorted to a more conservative solution (which seems okay in this context!). Good job!